csharp_dotnetcore/14.NLP-With-Dispatch/appsettings.json

```diff
@@ -0,0 +1,5 @@
+{
+  //  The MicrosoftAppId and MicrosoftAppPassword is created from the Azure portal http://portal.azure.com when you created your Bot.
```

Comment:
+1 we should not check in appsettings.json or .env files. the best practice is to gitignore these. 

Also in case you are going to do this, then the only two settings in the appsettings.json should be botFilePath and botFileSecret.